app/models/concerns/course/assessment/questions_concern.rb

```diff
@@ -17,7 +17,23 @@ def attempt(submission)
                          map { |answer| [answer.question, answer] }.to_h
 
     map do |question|
-      attempting_answers.fetch(question) { question.attempt(submission) }
+      attempting_answers[question] || question.latest_correct_answer(submission) ||
```

Comment:
I'm a little concerned about this because that changes the behaviour of attempt to not attempt a question if there already is a correct answer. That means, after running attempt, there might not be an answer that is editable.

Maybe we can have another method that checks for graded answers, attempting answers, then actually attempt the question?